lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -2695,6 +2691,12 @@ FuncInfo* PostVisitFunction(ParseNode* pnode, ByteCodeGenerator* byteCodeGenerat
         return top;
     }
 
+    if (top->paramScope && top->paramScope->GetCanMergeWithBodyScope())
+    {
+        Scope::MergeParamAndBodyScopes(pnode);
+        Scope::RemoveParamScope(pnode);
+    }
```

Comment:
Is this a change to how we handle the param scope that was necessary for precise capture?

I'm concerned that this may have bugs and if it is not specific to precise capture, would be better to have in a separate change.